### PR TITLE
Import sort with Prettier and Pre Commit with Husky

### PR DESCRIPTION
**Pull request by max-programming**
Continuation of #55 and also added Pre Commit Git Hooks that will format the code once it gets committed

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```
Comment (steven-tey) [inline .husky/pre-commit:1]: I'm probably missing something here but why is this file needed? Can we remove it?

**Review by steven-tey — CHANGES_REQUESTED** [review]
Thanks a lot for your work on this! However, I'm not sure how I feel about all the spaces that it's creating in the code base (refer https://github.com/trivago/prettier-plugin-sort-imports#output)

Is it possible to remove all spaces?

Also, is it possible to make this script run on every CMD+S, HMR-style?

**Comment by max-programming** [comment]
@steven-tey by spaces, do you mean the separation between those imports?

**Comment by max-programming** [comment]
On it 🔥

**Comment by max-programming** [comment]
@steven-tey Done with changes

**Comment by steven-tey** [comment]
Sorry for being a bit annoying with this but is it possible to remove all changes except for the prettier file, and vscode ones? I wanna run Prettier locally to make sure everything looks good before merging 😅

**Comment by max-programming** [comment]
Sure thing

**Comment by max-programming** [comment]
Looks like it's quite hard to go through more than 90 files and change them.

**Comment by max-programming** [comment]
@steven-tey Conflicts will increase if we don't deal with this now 😄

**Comment by steven-tey** [comment]
@max-programming could you create a new PR and commit only the `.prettierrc.json` and the `.vscode` config file? Don't run prettier locally please (and don't put it in `package.json` yet), I'll double check that and run it before merging. Thank you!

**Comment by max-programming** [comment]
@steven-tey So don't you want the `format:write` script and husky in the new PR?

**Comment by max-programming** [comment]
I hope #76 works. Let me know if you need more than that

**Pull request closed without merging**